test/common/README.md

```diff
@@ -111,8 +111,11 @@ Indicates if there is more than 1gb of total memory.
 ### expectWarning(name, expected)
 * `name` [&lt;string>]
 * `expected` [&lt;string>] | [&lt;Array>]
+* `code` [&lt;string>]
 
-Tests whether `name` and `expected` are part of a raised warning.
+Tests whether `name`, `expected`, and 'code' are part of a raised warning. If
+an expected warning does not have a code then common.noWarnCode can be used
```

Comment:
Nit: `common.noWarnCode` -> `` `common.noWarnCode` ``?

Reply:
Will fix this too, thanks.